extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -31,12 +33,14 @@ export class AmpAutoAds extends AMP.BaseElement {
     const type = this.element.getAttribute('type');
     user().assert(type, 'Missing type attribute');
 
-    const configUrl = getConfigUrl(type, this.element);
-    if (!configUrl) {
+    const adNetwork = getAdNetworkConfig(type, this.element);
+    if (!adNetwork) {
       return;
     }
-    this.getConfig_(configUrl).then(() => {
-      // TODO: Use the configuration to place ads.
+    this.getConfig_(adNetwork.getConfigUrl()).then(configObj => {
```

Comment:
Correct. Non-credentailed requests can be done earlier. We agreed to extensively comment this in the code to make it explicit.